server/games/coop.py

```diff
@@ -31,6 +33,35 @@ async def validate_game_mode_settings(self):
             "TeamSpawn": ("fixed", ValidityState.SPAWN_NOT_FIXED),
             "RevealedCivilians": ("No", ValidityState.CIVILIANS_REVEALED),
             "Difficulty": (3, ValidityState.WRONG_DIFFICULTY),
-            "Expansion": (1, ValidityState.EXPANSION_DISABLED),
+            "Expansion": ("true", ValidityState.EXPANSION_DISABLED),
```

Comment:
It would probably be nice if `1`, `"true"`, and `"True"` would all be accepted.

Reply:
Done. I made a generic boolean comparison that will handle true/false, yes/no, on/off, 1/0.